content/awell-orchestration/api-reference/queries/get-published-pathways.mdx

```diff
@@ -32,11 +32,27 @@ query GetPublishedPathwayDefinitions {
           value
         }
       }
+      track_definitions {
+        id
+        title
+        can_trigger_manually
+      }
+      dataPointDefinitions {
+        # @deprecated use data_point_definitions
+        id
+        # ...
+      }
     }
   }
 }
 ```
 
+## can_trigger_manually
```

Comment:
```suggestion
## Can trigger manually
```

Reply:
This doesn't make sense as a phrase, it's the name of the field this section is about
I can change the header to 'What does can_trigger_manually mean?' or 'The can_trigger_manually field' or 'How can I tell if a track can be added or scheduled to a flow?'

Reply:
Let's do "The can_trigger_manually field"